packages/apps/tools/src/pages/faucet/existing/index.tsx

```diff
@@ -128,6 +133,16 @@ const ExistingAccountFaucetPage: FC = () => {
         <Breadcrumbs.Item>{t('Existing')}</Breadcrumbs.Item>
       </Breadcrumbs.Root>
       <Heading as="h4">{t('Add Funds to Existing Account')}</Heading>
+      <div className={notificationContainerStyle}>
+        {showNotification ? (
+          <Notification.Root
+            color="warning"
+            expanded={true}
+            icon="Information"
+            title={t('The faucet can not be used on Mainnet. You can switch networks in the top bar.')}
```

Comment:
I still think we should display a different message. This one is kind of incorrect, as in, the network toggle has no effect on the Faucet, it just always executes on Testnet. So perhaps better might be, when on Mainnet, to show a notification like;

`The Faucet only runs on Testnet. The Network toggle in the topbar has no effect on the Faucet. It will not fund accounts on Mainnet, only on Testnet.`

Perhaps something along those lines, what do you think?

Reply:
Sound fine, yeah. I will change the text to this version